version_switcher/builders.py

```diff
@@ -0,0 +1,16 @@
+from sphinx.builders.dummy import DummyBuilder
+import json
+from .utils import load_versions_map, VERSION_MAP_FILE
+
+
+class OpenwispDummyBuilder(DummyBuilder):
+    name = 'ow_dummy'
```

Comment:
can you add a docstring explaining what this class does?
Can we find a better name for this? Like `VersionSwitcherIndexBuilder` or anything similar?